bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>"Now we have to find the numbers of our segments, ulna and humerus, in order to build the rototranslation matrices." </p><p> </p><p>Put this sentence after the results : "Ulna is segment 30 ans Humerus is segment 23."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>